kafka/build.yml

```diff
@@ -1,5 +1,9 @@
 repository: monasca/kafka
 variants:
+  - tag: 0.9.0.1-2.11-1.1.0
+    args:
+      KAFKA_VERSION: "0.9.0.1"
+      SCALA_VERSION: "2.11"
   - tag: 0.9.0.1-2.11-1.0.2
```

Comment:
@timothyb89 is it like an "official approach"- we should drop older entries in ```build.yml``` to disallow overriding older images with changes meant for newer ones ?